<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmyWidget</class>
 <widget class="QWidget" name="QmyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MetaObject的使用</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>22</y>
     <width>300</width>
     <height>302</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labBoy">
        <property name="text">
         <string>设置男生年龄</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoy">
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnBoyInc">
        <property name="text">
         <string>boy长大一岁</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnClassInfo">
        <property name="text">
         <string>类的元对象信息</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labGirl">
        <property name="text">
         <string>设置女生年龄</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinGirl">
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnGirlInc">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>girl长大一岁</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空文本框</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
